src/Defs/Alt.cpp

```diff
@@ -870,10 +870,12 @@ namespace lua::Class
 
 	int Alt::SetCursorPosition(lua_State* L)
 	{
-		alt::Vector2i position;
+		alt::Vector2f position;
+		bool normalized;
 
 		ArgumentReader argReader(L);
 		argReader.ReadVector(position);
+		argReader.ReadBoolDefault(normalized, false);
```

Comment:
you don't use the `normalized` variable when setting the cursors's position

Reply:
Both SetCursorPosition and GetCursorPosition accept normalized bool as seen in the C++ SDK [here](https://github.com/altmp/cpp-sdk/blob/b644b624308b54d3edc67aef91cc8b3cca2ffe25/ICore.h#L135)

Reply:
yes, but you don't use the variable here: https://github.com/drakeee/altv-lua-module/blob/36e7e5018d659fa6d5be5ce3cf9f4ea584a44c81/src/Defs/Alt.cpp#L886